service/src/main/java/uk/gov/hmcts/reform/fpl/service/HearingBookingService.java

```diff
@@ -50,4 +52,19 @@ public HearingBooking getHearingBooking(final List<Element<HearingBooking>> hear
             .map(Element::getValue)
             .orElse(HearingBooking.builder().build());
     }
+
+    public HearingBooking getHearingBookingByUUID(List<Element<HearingBooking>> hearingBookings, UUID elementId) {
+        return hearingBookings.stream()
+            .filter(hearingBookingElement -> hearingBookingElement.getId().equals(elementId))
+            .map(Element::getValue)
+            .findFirst()
+            .orElse(HearingBooking.builder().build());
+    }
+
+    public HearingDateDynamicElement getHearingDynamicElement(DynamicList list) {
+        return HearingDateDynamicElement.builder()
+            .date(list.getValue().getLabel())
+            .id(list.getValue().getCode())
+            .build();
```

Comment:
I am not sure if I like this here, maybe this could be inside HearingDateDynamicElement, like "fromDynamicList"

I don't think adding this to the service gives us value, whyat do you think?

Reply:
I had this here but was asking to move this to HearingBookingService. @Gogetter thoughts?